### PR TITLE
Fix "NoneType has no last_modified_by attribute" bug

### PR DESCRIPTION
**Pull request by SnaKyEyeS**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -191,8 +191,16 @@ def when_user_logged_out(sender, user):
...
+    if user.last_schedule_id is not None:
+        if session.get("current_schedule") is None:
+            session["current_schedule"] = user.get_schedule(
+                id=user.last_schedule_id
+            ).data
+
+        elif session["current_schedule"].is_empty():
+            session["current_schedule"] = user.get_schedule(
+                id=user.last_schedule_id
+            ).data
```
Comment (jeertmans) [inline app.py:203]: Shouldn't we be consistent about using __getitem__ (i.e. : session[...]) or get ?

File: views/calendar.py [hunk 2]
```diff
@@ -302,7 +302,13 @@ def save():
 def download():
     mng = app.config["MANAGER"]
     link = request.args.get("link")
-    choice = int(request.args.get("choice")) if request.args.get("choice") else 0
+
+    try:
+        choice = int(request.args.get("choice")) if request.args.get("choice") else 0
+    except ValueError as e:
+        choice = 0
+        # No comment... long live the user !
```
Comment (jeertmans) [inline views/calendar.py:310]: Maybe write a more meaningful comment :)

**Review by jeertmans — CHANGES_REQUESTED** [review]

File: app.py (hunk 1 above)
Comment (SnaKyEyeS) [reply app.py:203]: I use .get(...) when I need to check if the item is inside or not (to avoid a KeyError). In the other cases, I use ["...."].

Comment (jeertmans) [reply app.py:203]: Then wouldn't it be better to use __hasattr_(obj, attr) or equivalent ?

Comment (SnaKyEyeS) [reply app.py:203]: There's maybe a better way indeed, but that's a modification for another day (should do it in the whole project).

**Review by jeertmans — APPROVED** [review]

**Pull request merged**